README.md

```diff
@@ -2,15 +2,15 @@
 
 ## Nitric Examples
 
-This is the Nitric Examples repository, it contains starter project templates in a variety of languages supported by the Nitric Framework.
+This repository serves as the Nitric Examples repository, housing starter example templates written in various languages supported by the Nitric Framework. Each example is designed to be deployed using Nitric for AWS, GCP, and Azure, or alternatively, can be run locally. The examples are organized within the [v1 directory](./v1/), which represents our latest version.
```

Comment:
```suggestion
This repository serves as the Nitric Examples repository, housing both starter templates for the languages supported by the Nitric Framework as well as a variety of example projects to show how Nitric can be used for different use cases. 

Each example is designed to be deployed using Nitric for AWS, GCP, and Azure, or alternatively, can be run locally. The examples are organized within the [v1 directory](./v1/), which represents our latest version.
```

Reply:
Is it possible to organize them so that the starter templates are in a different folder from the example apps? I think it would be easier to find them if they're on their own?